### ISSUE TITLE
Black Screen with ARToolkitX on Android and Unity

### ISSUE THREAD

**Issue by nickos64252** [artoolkitx/arunityx#14]
Hello everyone,

I try to use ARToolkitX on Unity with ARUnityX.
I have Unity 2017.4.15f1 on Windows 10.

I downloaded the last version of ARUnityX (this git) and folowed the instructions to build it for windows and android.
I ended copying some files from the Source folder to my Unity project. Here are the files I copied : 
- Folder "artoolkitX-Unity"
- Folder "StreamingAssets"
- In Plugins : 

![filesx](https://user-images.githubusercontent.com/20517585/49031420-273f6300-f1aa-11e8-92f8-395f2d6891fc.png)

When I open SimpleScene and run it in Windows, it works fine.
When I run it on ANdroid, the screen stays black, and in Android monitor, I can see : 

"Ok - True .... Got Frame - False"
It looks like writing it each frame, it makes me think the program tries to access camera but fails.

Here is my manifest (I changed the package name to "com.artoolkitx.testUnity" in Unity) : 

![manifest](https://user-images.githubusercontent.com/20517585/49031672-e8f67380-f1aa-11e8-971d-6c0437753292.png)

Any ideas ?

Thank you

**Comment by nickos64252** [comment]
After some research, I found an error when starting. I don't know if it is related.
![log](https://user-images.githubusercontent.com/20517585/49073648-36b6be80-f233-11e8-9695-f0c9e49207c7.png)

**Comment by nickos64252** [comment]
Hello,
I think my manifest was wrong. The activity in application must be a "org.artoolkitx.arx.unity.ARUnityXPlayerActivity". 
I corrected my manifest to : 
![manifest2](https://user-images.githubusercontent.com/20517585/49087244-c9b52000-f256-11e8-8845-eb415a4b8644.png)

And now it looks like it cannot find this activity at all when the program starts. 
![log2](https://user-images.githubusercontent.com/20517585/49087318-f23d1a00-f256-11e8-83c0-82448a0e948b.png)

But I may be wrong on all of that.

**Comment by nickos64252** [comment]
I think it is due to problems in my plugins for android. Can someone send me the compiled plugins ?
Thanks

**Comment by nickos64252** [comment]
After some research, I found out the Unity Android plugin did not compile (with "./build.sh android") due to a problem in my JDK.
I can now compile it. It generates "arunityXPlayer-release.aar".
I wanted to use the packaging command, but there are Mac command (line56) in it. So I just copied the above file to my Plugin folder.
I now have an image on Android, but the recognition doesn't work.

Error : "CameraEventListenerUnityImpl :  cameraStarted::arwAndroidVideoPushInit() failed"

I think my manifest and plugin are ok, so I don't know what is the error.
